docker/base-images/Dockerfile.osmt-base

```diff
@@ -1,4 +1,4 @@
-FROM centos:centos8.3.2011 as osmt-base
+FROM rockylinux:9.0.20220720 as osmt-base
```

Comment:
Based on this...
- https://pythonspeed.com/articles/centos-8-is-dead/

... I'm inclined to go with RedHat's Universal Base Images (UBI's), or Oracle's RedHat clone images.

Reply:
made the switch to oracle linux base image. the base image has been tested and works fine